### DISCUSSION TITLE
loading tailwind full color palette

### DISCUSSION THREAD

**Discussion by sandeep-gh** [elimintz/justpy#415] [Q&A]
How can I load (prefrebly from justpy) the full color palette? Typically, there is tailwind.config.js that needs to be modified [1]
. But can't seem to find that in justpy codebase. 

Can I add the config snippet in loca/templates/tailwind.html?

1. https://gist.github.com/sebmellen/e939ce8a29fd6cb611ef6f283f700f2c

**Comment by elimintz** [comment]
Justpy comes out of the box with the CDN version of Tailwind. What you can do, is use npm to generate a css file to your liking and then add it to your pages using the css attribute: https://justpy.io/reference/webpage/#css

When you do this, don't forget to create your WebPages with tailwind=False or add TAILWIND=False to justpy.env:

```python
def request_handler(request):
   wp = jp.WebPage(tailwind=False, css=my_css)
   .....
   return wp

```
